Apps/Examples/Examples/All Examples/ShowHideLayerExample.swift

```diff
@@ -95,7 +95,7 @@ class ShowHideLayerExample: UIViewController, ExampleProtocol {
         // Update the museum layer's visibility based on whether the switch
         // is on. `visibility` is `nil` by default.
         do {
-            try style.updateLayer(withId: museumLayerId) { (layer: inout CircleLayer) in
+            try style.updateLayer(withId: museumLayerId, type: CircleLayer.self) { (layer: inout CircleLayer) in
```

Comment:
Is it possible to remove the type from the closure argument list now?

```suggestion
            try style.updateLayer(withId: museumLayerId, type: CircleLayer.self) { (layer) in
```

Reply:
If so, let's apply that change throughout this PR